### PR TITLE
[FIX] l10n_ar: improve i18n of "Registered Accountants" in COA's name

### PR DESCRIPTION
**Pull request by sovi-odoo**
The old name caused caused some confusion from clients

- Old: "Plan Contable Generico Argentino para Contadores Publicos"
- New: "Plan Contable Genérico Argentino para Responsables Inscriptos"

task-3921529

### CODE REVIEW

**Review by malb-odoo — APPROVED** [review]
LGTM thanks for this ! Though i would say it's more of a FIX since we are doing it in stable :smile: @sovi-odoo

File: addons/l10n_ar/i18n/es_419.po [hunk 1]
```diff
@@ -346,7 +346,7 @@ msgstr "Plan Contable Genérico Argentino para Personas Físicas Exentas"
 #: code:addons/l10n_ar/models/template_ar_ri.py:0
 #, python-format
 msgid "Argentine Generic Chart of Accounts for Registered Accountants"
-msgstr "Plan Contable Genérico Argentino para Contadores Públicos"
+msgstr "Plan Contable Genérico Argentina para Responsables Inscriptos"
```
Comment (vbe-odoo) [inline addons/l10n_ar/i18n/es_419.po:349]:
```suggestion
msgstr "Plan Contable Genérico Argentino para Responsables Inscriptos"
```

**Review by vbe-odoo — COMMENTED** [review]
@sovi-odoo @malb-odoo just a small change.

**Comment by vbe-odoo** [comment]
@sovi-odoo @malb-odoo thanks for the changes and review.
This is ready for final review/merge.
Thanks!
cc' @ren-odoo @fvz-odoo @masi-odoo

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**